extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1226,6 +1226,8 @@ public static function getCurrentStagingHost( $dbName='', $default='', $host = n
 	static public function getLocalEnvURL( $url, $forcedEnv = null ) {
 		global $wgWikiaEnvironment, $wgWikiaBaseDomain, $wgDevDomain;
 
+        Wikia::log( __METHOD__, "", "WikiFactory is not used." );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace or remove this deprecated Wikia::log logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)